gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=io.ballerina
-version=0.5.1-SNAPSHOT
+version=0.6.0-SNAPSHOT
 
 #dependency
 ballerinaLangVersion=2201.5.0
```

Comment:
Don't we need to update the ballerina lang timestamped version?

Reply:
No need right. Since this doesnt depend on the latest lang changes we dont need to

Reply:
But the other stdlibs timestamps are released on top of the lang update 6 timestamp IINM. so shouldn't we update the lang timestamp as well?

Reply:
Other stdlibs have not update their lang versions as well.